components/mediation-initializer/org.wso2.carbon.mediation.light.initializer/src/main/java/org/wso2/carbon/mediation/initializer/ServiceBusInitializer.java

```diff
@@ -488,7 +488,7 @@ public void createSynapseDebugEnvironment(ServerContextInformation contextInfo)
             contextInfo.setSynapseDebugInterface(debugInterface);
             SynapseDebugManager debugManager = SynapseDebugManager.getInstance();
             contextInfo.setSynapseDebugManager(debugManager);
-            log.info("Synapse debug Environment created successfully");
+            log.debug("Synapse debug Environment created successfully");
```

Comment:
better if check  if (log.isDebugEnabled()) {

Reply:
Isn't it ok since we do not have any string concatenation?